tdrs-frontend/src/reducers/requestAccess.js

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
This is the reducer for `requestAccess` which will set `loading` and `requestAccess`.